CMakeLists.txt

```diff
@@ -72,6 +72,9 @@ if (NOT PSTORE_IS_INSIDE_LLVM)
 endif ()
 set (PSTORE_ROOT_DIR "${CMAKE_CURRENT_SOURCE_DIR}")
 
+# Define pstore source and binary dirs similar to the definitions in clang
+set(PSTORE_SOURCE_DIR ${CMAKE_CURRENT_SOURCE_DIR})
```

Comment:
Don’t forget the space before the open-parenthesis. Is there a difference between `PSTORE_SOURCE_DIR` and `PSTORE_ROOT_DIR` (above)? Should there be quotes around the use of `CMAKE_CURRENT_SOURCE_DIR`?

Reply:
> Don’t forget the space before the open-parenthesis

(This also applies elsewhere.)

Reply:
> Don’t forget the space before the open-parenthesis.

Changed.

> Is there a difference between PSTORE_SOURCE_DIR and PSTORE_ROOT_DIR (above)?

PSTORE_ROOT_DIR is the form used in LLVM and Clang so is more consistent and clearer in the context that it is used here.

> Should there be quotes around the use of CMAKE_CURRENT_SOURCE_DIR?

I don't believe they are required.